resources/problem-vote.js

```diff
@@ -0,0 +1,91 @@
+var voteChart = null;
+
+function init_problem_vote_form() {
+    $('#delete-problem-vote-form').on('submit', function (e) {
+        e.preventDefault();
+        $.ajax({
+            url: $('#delete-problem-vote-form').prop('action'),
+            type: 'POST',
+            data: $('#delete-problem-vote-form').serialize(),
+            success: function () {
+                $('#problem-vote-button').text(gettext('Vote on problem points'));
+                $.featherlight.close();
+            },
+            error: function (data) {
+                alert(interpolate(gettext('Unable to delete vote: %s'), [data.message]));
```

Comment:
`data.message` -> `data.responseJSON.message`

Reply:
Is this necessary? The earlier code didn't use `responseJSON` when looking up `points`.

Reply:
Yes, the error path no longer alerts. I think it's because `DeleteProblemVote.post` returns a `JsonResponse`.

Reply:
done